Plugins/BuildNotifications.Plugin.Tfs/TfsConnectionPool.cs

```diff
@@ -83,24 +88,7 @@ private VssCredentials CreateCredentials(TfsConfiguration data)
             return new VssCredentials(new VssBasicCredential(username, pw?.PlainText()));
         }
 
-        private bool IsAuthenticatedId(Guid authenticatedIdentityId)
-        {
-            return !authenticatedIdentityId.Equals(Guid.Parse("aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa"));
-        }
-
-        private bool IsOnPremiseServer(string? url)
-        {
-            if (string.IsNullOrWhiteSpace(url))
-                return false;
-
-            var uri = new Uri(url);
-            return uri.Host != "dev.azure.com";
-        }
-
-        private bool NeedsToAppendCollectionName(TfsConfiguration data)
-        {
-            return IsOnPremiseServer(data.Url);
-        }
+        private bool IsAuthenticatedId(Guid authenticatedIdentityId) => !authenticatedIdentityId.Equals(Guid.Parse("aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa"));
```

Comment:
Todo implement? Or removable?

Reply:
Haha... iirc this is the DevOps version of an empty guid...
I'll check this and add a comment if this is really the case.

Reply:
Jep. That's the ID of the "Anonymous" User. Extracted named constant for this